windows/windows_subsystem_for_linux/install_wsl.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+# Install inbox version of WSL
+# Parameters:
```

Comment:
These tasks files only used in current test case so no need to specifiy 'Parameters'.

Reply:
Fixed.